article/app/layout/LiveBlogs.scala

```diff
@@ -0,0 +1,32 @@
+package layout
+
+import java.util.Locale
+
+import common.Edition
+import model.Content.BodyBlocks
+import model.Content.BodyBlocks.{KeyEvent, UnclassifiedEvent, SummaryEvent}
+import org.joda.time._
+import org.joda.time.format._
+import play.api.mvc.RequestHeader
+
+object BlockFields {
+
+  def apply(block: BodyBlocks) = block.eventType match {
+    case SummaryEvent => " is-summary"
+    case KeyEvent => " is-key-event"
+    case UnclassifiedEvent => ""
```

Comment:
yes just because I'm being lazy and using string concatentation rather than mkString